### PR TITLE
[scripts] Minor improvements to `macdeployqtplus` script.

### PR DESCRIPTION
**Pull request by chrisgavin**

### CODE REVIEW

File: contrib/macdeploy/macdeployqtplus [hunk 1]
```diff
@@ -302,7 +302,6 @@ def copyFramework(framework, path, verbose):
         if os.path.exists(fromContentsDir):
             toContentsDir = os.path.join(path, framework.destinationVersionContentsDirectory)
             shutil.copytree(fromContentsDir, toContentsDir, symlinks=True)
-            contentslinkfrom = os.path.join(path, framework.destinationContentsDirectory)
```
Comment (jonasschnelli) [inline contrib/macdeploy/macdeployqtplus]: What's the reason to remove this line? We don't use dynamic linked custom Frameworks in the Bitcoin-Qt.app bundle.

Comment (chrisgavin) [inline contrib/macdeploy/macdeployqtplus]: In af0bd5e, the majority of this code was removed but I think this line was left in by mistake. As far as I can tell it doesn't do anything - it just defines a variable that is never used.

**Comment by jonasschnelli** [comment]
Yes. Makes sense (though a very tiny optimisation/cleanup near PR "dust-limit").
utACK e8babc4330cd7db82650f28147c388846a389c77

**Comment by jonasschnelli** [comment]
Builds the DMG correctly (https://bitcoin.jonasschnelli.ch/build/113).
Tested ACK e8babc4330cd7db82650f28147c388846a389c77

**Pull request merged**